src/deck-options.md

```diff
@@ -386,35 +346,32 @@ The options are:
+time in "Seconds to show question for" and/or "Seconds to show answer for". Then, in the
 review screen, use the Auto Advance action from the `More` button to start advancing.
 
 ## Burying
 
-When Anki gathers cards, it first gathers intraday learning cards, then interday learning cards, then reviews, and finally new cards. This affects how burying works:
+When Anki gathers cards, it first gathers intraday learning cards, then interday learning cards, then review cards, and finally new cards. This affects how burying works:
 
 - If you have all burying options enabled, the sibling that comes earliest in that list will be shown. For example, a review card will be shown in preference to a new card.
 - Siblings later in the list can not bury earlier card types. For example, if you disable burying of new cards, and study a new card, it will not bury any interday learning or review cards, and you may see both a review sibling and new sibling in the same session.
 
 The options are:
 
-- Bury new siblings: whether other new cards of the same note (e.g., reverse cards, adjacent cloze deletions) will be delayed until the next day.
-- Bury review siblings: whether other review cards of the same note will be delayed until the next day.
-- Bury interday learning siblings: whether other learning cards of the same note with intervals >= 1 day will be delayed until the next day.
+- **Bury new siblings**: Whether other new cards of the same note (e.g. reverse cards, adjacent cloze deletions) will be delayed until the next day.
+- **Bury review siblings**: Whether other review cards of the same note will be delayed until the next day.
+- **Bury interday learning siblings**: Whether other learning cards of the same note that crossed a day boundary will be delayed until the next day.
 
 For more info about burying cards, please see [this section](./studying.md#siblings-and-burying) of the manual.
 
 ## Audio
 
-By default, Anki automatically plays audio on the front and back of
-cards. If you check _Don't play audio automatically_, Anki will not play
-audio until you press the replay audio key, `r` or `F5`.
+- **Don't play audio automatically**: By default, Anki automatically plays any audio you have on
+cards. If you turn on this option Anki will not play audio until you press the replay audio key, <kbd>r</kbd> or <kbd>F5</kbd>.
 
-_Always include question side when replaying audio_ controls whether audio from
-the question side should be played when replaying the audio while an answer is
-shown. Please note that it does not control what happens when you show the
+- **Skip question when replaying answer**: Controls whether audio from
+the question side should be played when replaying the audio while on answer side. Please note that it does not control what happens when you show the
 answer; for that please see [this section](templates/fields.md#special-fields).
```

Comment:
Anybody knows what is this cross-referencing? I didn't find anything.

Reply:
Isn't this on point? Replay is controlled with that Deck Options setting -- initial-play is controlled on the template. 
> FrontSide will not automatically play any audio that was on the front side of the card. If you wish to have the same audio play automatically on both the front and back of the card, you’ll need to manually include the audio fields on the back as well.

Reply:
I was thrown into some unfamiliar section; didn't even noticed that line. I changed this to:

> Controls whether audio from the question side is played when you use replay action on the answer side. Note that, Anki does not automatically play audio from the {{FrontSide}} field. This option does not influence the behaviour of automatic play.